### DISCUSSION TITLE
ASK: Should ARM PC's be supported right now?

### DISCUSSION THREAD

**Discussion by randyrants** [microsoft/winget-cli#321] [General]
# Description of the new feature/enhancement

Better support for Surface Pro X and/or ARM

# Proposed technical implementation details (optional)

It may be that this is already supported and simply how packages are being uploaded to the catalog, but I'm having a painful time trying to get anything to run on a Surface Pro X.  For apps that only have x64 installed, that would make sense but I know that some of these app are supported on Pro X, like Discord.Discord.  

Ideally, if there is no ARM version available then it would try an x86 version but what I'm seeing is:
```
C:\>winget install -e Discord.Discord
Found Discord [Discord.Discord]
No installers are applicable to the current system
```

**Comment by megamorf** [comment]
That's not possible yet, so subscribe to #132 for updates.

**Comment by randyrants** [comment]
> That's not possible yet, so subscribe to #132 for updates.

Subscribed - thanks!

**Reply by pb5050** [reply]
any updATE On this>


No installers are applicable to the current system.
No installers are applicable to the current system.
No installers are applicable to the current system.
No installers are applicable to the current system.

for 90% of files i find. i am using surface pro X.

would be nice if -force would bypass that. since it emulating X64 and X86   and when using force it should say wat ARCH the file we are about to install is.

**Comment by denelon** [comment]
We are actively discussing support for ARM architecture. Enhancements coming with the new manifest schemas will help address this gap. We will still have some work to identify when emulation is supported on a machine, and improvements need to be made for search to identify packages that are valid for different architectures.
